dcrwallet.go

```diff
@@ -137,6 +137,9 @@ func walletMain() error {
 			startPromptPass(w)
 		}
 		startWalletRPCServices(w, rpcs, legacyRPCServer)
+		if w.StakeMiningEnabled {
+			go startTicketPurchase(w)
```

Comment:
well in the gRPC server there is no notion of an unlocked wallet. the private passphrase is passed as a field in every RPC request message that requires it.  And we want this usable over grpc